src/System.Management.Automation/engine/Interop/Windows/GetDosDevice.cs

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#nullable enable
+
+#if !UNIX
+using System;
+using System.Buffers;
+using System.ComponentModel;
+using System.Management.Automation;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Windows
```

Comment:
```suggestion
    internal static partial class Kernel32
```

Reply:
https://grep.app/search?q=partial%20class%20Kernel32&words=true
710 results

https://grep.app/search?q=partial%20class%20Windows&words=true
20 results

Reply:
We have too little of pinvokes to create separate classes.